tmt/utils.py

```diff
@@ -368,8 +368,504 @@ def to_spec(self) -> dict[str, Any]:
         return dict(self)
 
 
-# A "environment" type, representing name/value environment variables.
-EnvironmentType = dict[str, str]
+#: A type of environment variable name.
+EnvvarName: 'TypeAlias' = str
+
+# This one is not an alias: a full-fledged class makes type linters
+# enforce strict instantiation of objects rather than accepting
+# strings where `EnvvarValue` is expected.
+
+
+class EnvvarValue(str):
```

Comment:
Could we CamelCase all words of the name?

```suggestion
class EnvVarValue(str):
```

Reply:
Changed in 4102fc69